### PR TITLE
linux/[h-k]: add more information link

### PR DESCRIPTION
**Pull request by schneiderl**
It's been a while since I don't contribute here on tldr. 

I'll be trying my best to finish #6062  in the next few days. 

In this PR, `kde-inhibit` was not covered because I did not found a decent source that we can use. Does anyone have suggestions?

### CODE REVIEW

File: pages/linux/hardinfo.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # hardinfo
 
 > Show hardware information in GUI window.
+> More information: <https://manned.org/hardinfo>.
```
Comment (marchersimon) [inline pages/linux/hardinfo.md:4]:
```suggestion
> More information: <https://github.com/lpereira/hardinfo>.
```

File: pages/linux/hwclock.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # hwclock
 
 > Used for reading or changing the hardware clock. Usually requires root.
+> More information: <https://man7.org/linux/man-pages/man8/hwclock.8.html>.
```
Comment (marchersimon) [inline pages/linux/hwclock.md:4]:
```suggestion
> More information: <https://manned.org/hwclock>.
```
Any reason you used the `man7.org` link here?

File: pages/linux/imgp.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # imgp
 
 > Command-line image resizer and rotator for JPEG and PNG images.
+> More information: <https://manned.org/imgp>.
```
Comment (marchersimon) [inline pages/linux/imgp.md:4]:
```suggestion
> More information: <https://github.com/jarun/imgp>.
```

**Review by marchersimon — COMMENTED** [review]
Thanks, I left a few suggestions below.

**Comment by schneiderl** [comment]
Do we prefer to use GitHub links whenever they are available? I'll do so on my following PRs then.

**Comment by marchersimon** [comment]
> Do we prefer to use GitHub links whenever they are available? I'll do so on my following PRs then.

We prefer any official source, except it doesn't provide any useful information. Most of the time I just look at the bottom of the `manned.org` link and see if there's any link to the home page of the command.

**Comment by marchersimon** [comment]
This is what I found for `kde-inhibit` :<https://invent.kde.org/plasma/kde-cli-tools/-/tree/master/kdeinhibit>.
(Maybe we could also use <https://linuxcommandlibrary.com/man/kde-inhibit> 😄)

**Comment by marchersimon** [comment]
Oh, almost forgot about that this one. @schneiderl, do you want to finish this PR or should we take over?

**Comment by schneiderl** [comment]
@marchersimon Sorry for the delay. I'm very busy lately. Isn't this ready to be merged?

**Comment by marchersimon** [comment]
@schneiderl my [comment](https://github.com/tldr-pages/tldr/pull/6227#issuecomment-880704193) above about `kde-inhibit` is still unresolved. But we can just do that one in another PR.

**Comment by marchersimon** [comment]
@sbrl, @CleanMachine1 any opinions about the link for `kde-inhibit`?

**Review by blueskyson — APPROVED** [review]
Looks good to me.

**Review by marchersimon — APPROVED** [review]
Let's just leave out `kde-inhibit` for now. Thanks @schneiderl

**Pull request merged**